roles/openshift_health_checker/library/etcdstats.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python
+# vim: expandtab:tabstop=4:shiftwidth=4
+'''
+Ansible module to determine the size of an etcd cluster up to a specified limit
+'''
+
+import etcd
+
+from ansible.module_utils.basic import AnsibleModule
+
+
+def get_size(client, key, total=0, limit=0, depth=0, depth_limit=1000, visited=None):
```

Comment:
@rhcarvalho added a depth limit here that defaults to `1000`, I cannot foresee a stable etcd setup having any of its subtrees with a bigger height than this, but feel free to add your thoughts